ProcessDashboard/Model/Data Access Layer/TimeLog.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace ProcessDashboard.Model.Data_Access_Layer
+{
+    class TimeLog
+    {
+        int rowID { get; set; }
+
+        String timeLogID { get; set; }
+        String taskID { get; set; }
+        DateTime startDate { get; set; }
+        long elapsedTime { get; set; }
+        long interruptTime { get; set; }
```

Comment:
FYI: at the moment, the dashboard rich client and the REST APIs will only track elapsed and interrupt time in integer minutes. If you store a value to the REST API with fractional minutes, you may receive an error back.